tests/state_test.go

```diff
@@ -99,6 +99,13 @@ func (suite *ExecutionSpecStateTestSuite) TestExecutionSpecState() {
 	st.skipLoad(`^cancun\/eip4844_blobs\/`)
 	// calculate the different consumed gas because 0x0a and 0x0b contract is set to access list by ActivePrecompiles in Cancun
 	st.skipLoad(`^prague\/eip2537_bls_12_381_precompiles\/bls12_precompiles_before_fork\/precompile_before_fork.json\/tests\/prague\/eip2537_bls_12_381_precompiles\/test_bls12_precompiles_before_fork.py::test_precompile_before_fork`)
+	st.skipLoad(`^prague\/eip2537_bls_12_381_precompiles\/`)
+	st.skipLoad(`^prague\/eip7702_set_code_tx\/`)
+	st.skipLoad(`^prague\/eip7623_increase_calldata_cost\/.*type_3.*`)
+
+	// temporary skip failing frontier tests
+	st.skipLoad(`^frontier\/opcodes\/all_opcodes\/all_opcodes.json`)
+	st.skipLoad(`^frontier\/precompiles\/precompile_absence\/precompile_absence.json`)
```

Comment:
```
	st.skipLoad(`^prague\/eip2537_bls_12_381_precompiles\/`)
	st.skipLoad(`^prague\/eip7702_set_code_tx\/`)
	// temporary skip failing frontier tests
	st.skipLoad(`^frontier\/opcodes\/all_opcodes\/all_opcodes.json`)
	st.skipLoad(`^frontier\/precompiles\/precompile_absence\/precompile_absence.json`)
```
Could you place these in 95 because these should be removed in the future.

```
st.skipLoad(`^prague\/eip7623_increase_calldata_cost\/.*type_3.*`)
```
Please append the comment of the reason why this test skip.

Reply:
Done, thanks. 28db4fc